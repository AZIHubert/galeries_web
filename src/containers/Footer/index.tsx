import * as React from 'react';

import Text from '#components/Text';

import {
  Footer as FooterContainer,
} from './styles';

const Footer = () => (
  <FooterContainer>
    <Text
      fontWeight='lighter'
      styles={{
        fontSize: 0.65,
        marginBottom: 10,
      }}
      stylesMobile={{
        fontSize: 0.7,
      }}
      stylesLaptopL={{
        fontSize: 0.9,
      }}
    >
      Allan Aoudji @2021
    </Text>
  </FooterContainer>
);

export default Footer;
