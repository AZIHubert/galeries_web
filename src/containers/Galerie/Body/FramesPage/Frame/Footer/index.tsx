import moment from 'moment';
import * as React from 'react';
import { AiOutlineHeart } from 'react-icons/ai';

import Text from '#components/Text';

import theme from '#helpers/theme';

import {
  Container,
  LikeContainer,
} from './styles';

interface FooterI {
  createdAt: Date;
  likes: LikeI[];
}

const Footer = ({
  createdAt,
  likes,
}: FooterI) => (
  <Container>
    <LikeContainer>
      <Text
        fontWeight='bold'
      >
        {likes.length} like{likes.length > 1}
      </Text>
      <AiOutlineHeart
        color={theme.colors.danger}
        size={25}
      />
    </LikeContainer>
    <Text
      fontWeight='lighter'
      styles={{
        fontSize: 0.55,
        textAlign: 'center',
      }}
    >
      {moment(createdAt).calendar()}
    </Text>
  </Container>
);

export default Footer;
