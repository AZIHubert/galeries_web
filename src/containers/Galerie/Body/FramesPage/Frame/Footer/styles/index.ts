export { default as Container } from './Container';
export { default as LikeContainer } from './LikeContainer';
