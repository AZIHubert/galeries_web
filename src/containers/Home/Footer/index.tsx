import * as React from 'react';

const Footer = () => (
  <footer>
    <p>
      Allan Aoudji @2021
    </p>
  </footer>
);

export default Footer;
