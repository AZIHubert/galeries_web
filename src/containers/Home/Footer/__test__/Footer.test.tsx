import * as React from 'react';
import renderer from 'react-test-renderer';

import Footer from '../index';

describe('Footer', () => {
  it('renders without crashing', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
