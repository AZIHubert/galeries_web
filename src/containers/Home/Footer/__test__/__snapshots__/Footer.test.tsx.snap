// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders without crashing 1`] = `
<footer>
  <p>
    Allan Aoudji @2021
  </p>
</footer>
`;
